tests/spec/Bolt/Menu/BackendMenuBuilderSpec.php

```diff
@@ -10,7 +10,6 @@
 use Bolt\Twig\ContentExtension;
 use Knp\Menu\FactoryInterface;
 use Knp\Menu\ItemInterface;
-use Knp\Menu\MenuItem;
```

Comment:
strange that ECS didn't pick this one up, earlier.. 🤔

Reply:
It was looking only into src folder